build/source/engine/derivforce.f90

```diff
@@ -195,19 +202,28 @@ subroutine derivforce(time_data,forc_data,attr_data,mpar_data,prog_data,diag_dat
  scalarCO2air = co2Factor * airpres  ! atmospheric co2 concentration (Pa)
  scalarO2air  = o2Factor * airpres   ! atmospheric o2 concentration (Pa)
 
+ ! determine timeOffset based on tmZoneInfo option`
+ select case(model_decisions(iLookDECISIONS%tmZoneInfo)%iDecision)
+  ! Time zone information from NetCDF file
+  case(ncTime)
+   timeOffset = longitude/360._dp - tmZoneOffsetFracDay ! time offset in days
```

Comment:
WHY IS THERE `longitude/360` HERE?

Reply:
SIMMER DOWN!

Reply:
Take a look at the `tmZoneInfo` description here: 

https://github.com/bartnijssen/summa/blob/d42a414539a0cd08d35e74089f39c2bcbb48528f/docs/configuration/SUMMA_model_decisions.md

(this one is easiest to read - you can also look at this file in the PR itself).

If I am not mistaken, all the solar calculations are done in local time and if I understand it correctly, we are making the `timeOffset` so that the forcings align with the local time (see also time zone discussion in PR #317). If that is not how we want to implement it, then we can do it differently, but I was following the logic that already was present (the location where the `timeOffset` was added in `derivforce`.

Reply:
Looking at the example may be the most informative.

Reply:
OK, looks good.